scripts/get-badges.ts

```diff
@@ -99,34 +90,41 @@ interface UsersWithVisitsResult {
   );
 
   // TODO: filter enteredVenueIds and visitsTimeSpent so that they only contain related venues?
-  const result = usersWithVisits.map((userWithVisits) => {
-    const { user, visits } = userWithVisits;
-    const { id, partyName, enteredVenueIds = [] } = user;
-    const { email } = authUsersById[id] ?? {};
-
-    const visitsTimeSpent = visits.map(
-      (visit) => `${visit.id} (${formatSecondsAsDuration(visit.timeSpent)})`
-    );
```

Comment:
We may want to keep this formatted version of the time spent. Or alternately, include another column that includes this, as it's quite user-friendly data to include.

Reply:
https://github.com/sparkletown/sparkle/pull/1167/commits/84b4c1a4a6bc5a9bd4dfd083287e9fac85ccd10f